pkg/containerd/events.go

```diff
@@ -206,6 +235,91 @@ func (e *EventMonitor) Close() error {
 	return finalErr
 }
 
+// When namespaced network is enable, if portbinding is bound to 127.0.0.1 we update the DNAT
+// in main CNI DNAT chain (CNI-HOSTPORT-DNAT). The existing DNAT rule are as follows:
+// DNAT       tcp  --  anywhere             localhost            tcp dpt:9119 to:10.4.0.22:80.
+// While, this rule only allows the traffic to get routed to the localhost only we update
+// the rule to allow the traffic to any destination IP address.
+// DNAT       tcp  --  anywhere             anywhere             tcp dpt:9119 to:10.4.0.22:80.
+func execIptablesRules(portMappings nat.PortMap, containerID, namespace, pid string) error {
```

Comment:
While the comment describes what this method is supposed to do, it doesn't relate to the inputs. This comment is probably more appropriate for `createLoopbackIPtablesRules` instead? That does mean `execIptablesRules` still needs a better comment, though.

Reply:
You are right, I ended up moving things around and left the comment behind.